### PR TITLE
Name detection - Format Structured value correctly when detection fails

### PR DESCRIPTION
**Pull request by chiragjn**

### CODE REVIEW

File: ner_v1/detectors/textual/name/name_detection.py [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
-from lib.nlp.pos import *
-from ner_v1.constant import FIRST_NAME, MIDDLE_NAME, LAST_NAME
-from ner_v1.detectors.textual.text.text_detection import TextDetector
-from ner_v1.constant import EMOJI_RANGES
+
 from language_utilities.constant import ENGLISH_LANG, HINDI_LANG
-from ner_v1.detectors.textual.name.hindi_const import  \
-    HINDI_BADWORDS, HINDI_QUESTIONWORDS, HINDI_STOPWORDS, NAME_VARIATIONS
 from lib.nlp.const import nltk_tokenizer
+from lib.nlp.pos import *
```
Comment (srajdev) [inline ner_v1/detectors/textual/name/name_detection.py]: F403 'from lib.nlp.pos import *' used; unable to detect undefined names

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Pull request merged**